### ISSUE TITLE
Get screenshots of the app for readme

### ISSUE THREAD

**Issue by cheshire137** [cheshire137/overwatch-team-comps#124]
Instead of having the mockups at the top of the readme, I'd like to include screenshots of the app in action. I'm thinking one of the composition form and one of the hero pool page. I don't think we need screenshots of the about page, settings, or the composition view.

/cc @RobThePM

**Comment by RobThePM** [comment]
agreed! i can try to supply the screenshots this weekend if no one else gets to it first.

**Comment by RobThePM** [comment]
@cheshire137 is this what you need?

![10](https://cloud.githubusercontent.com/assets/26153402/26611136/fde7fc84-4560-11e7-89b1-1ec89da55fc7.png)

![sample - hero pool 02](https://cloud.githubusercontent.com/assets/26153402/26611177/4fab887e-4561-11e7-845c-c981a76f6ef1.png)
